integration/monai/examples/spleen_ct_segmentation/README.md

```diff
@@ -22,29 +22,75 @@ pip install -r virtualenv/requirements.txt
 
 ### 2. Download the Spleen Bundle
 ```
-python -m monai.bundle download --name "spleen_ct_segmentation_v0.1.1" --bundle_dir ./job/app/config
+python3 -m monai.bundle download --name "spleen_ct_segmentation_v0.1.1" --bundle_dir ./job/app/config
 ``` 
 
 ### 3. Download the data
 Download the spleen CT data from the [MSD challenge](http://medicaldecathlon.com/) and update data path.
 
 > **Note:** The dataset will be saved under `./data`. 
 ```
-python download_spleen_dataset.py
+python3 download_spleen_dataset.py
 sed -i "s|/workspace/data/Task09_Spleen|${PWD}/data/Task09_Spleen|g" job/app/config/spleen_ct_segmentation/configs/train.json
 ```
 
-### 4. Run NVFlare in POC mode
+### 4. Run experiment in simulator
+
+#### 4.1 Single thread, single gpu
+In resource restricted environments where you need to simulate several clients (-n 2 in this case) on the same GPU device, 
+you can run the simulator using:
+
+```
+python3 -u -m nvflare.private.fed.app.simulator.simulator job -o /tmp/nvflare/sim_spleen_ct_seg --thread 1 -n 2
+```
+
+#### 4.2 Multiple threads, multiple gpus
+If you have several gpus in your system, you can assign one for each client and use two threads. 
+We can also specify the client names via the `--client_list` argument 
+and assign them to the appropriate GPU device using the `--gpu` argument.
+
+```
+python3 -u -m nvflare.private.fed.app.simulator.simulator job -o /tmp/nvflare/sim_spleen_ct_seg --thread 2 --client_list site-1,site-2 --gpu 0,1
+```
+
+### 4.3 TensorBoard visualization
```

Comment:
should this be `####` to be consistent with 4.1?

Reply:
Thanks for noticing. I changed it.